src/domains/event/dto/index.ts

```diff
@@ -185,3 +185,21 @@ export class UpdateTimeTracking {
     this.endTime = input.endTime;
   }
 }
+
+export class ManualTimeModificationDTO {
```

Comment:
Missing JS Doc here for the class and the attributes.

Reply:
added